webhookserver/chart/templates/server.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       - name: stackrox
       - name: stackrox-dockerhub
       containers:
-      - image: quay.io/rhacs-eng/webhookserver:1.2
+      - image: quay.io/rhacs-eng/qa-multi-arch:webhookserver-1.2
```

Comment:
new multiarh image is broken working on upload a new image

Reply:
Is the multiarch good to go now?

Reply:
Yes the image should be ready now